tests/baselines/reference/additionOperatorWithNullValueAndInvalidOperator.errors.txt

```diff
@@ -23,37 +23,37 @@ tests/cases/conformance/expressions/binaryOperators/additionOperator/additionOpe
     
     // null + boolean/Object
     var r1 = null + a;
-             ~~~~~~~~
-!!! error TS2365: Operator '+' cannot be applied to types 'boolean' and 'boolean'.
+             ~~~~
+!!! error TS2531: Object is possibly 'null'.

```

Comment:
Two orthogonal issues:

First, we could do extra work to omit the word `possibly` when the type is *exactly* `null` or `undefined`. However, it is very rare for someone to *explicitly* use `null` or `undefined` as an operand, so it's not clear that it really matters all that much. This is the type of error that only surfaces in a test.

Second, we could consider using the word `value` instead of `object` when reporting for expression operands--but it's really the same error that we use for `obj.foo` where `obj` may be null or undefined. The downside would be error message searchability, but since the error is pretty obvious that may not be an issue.